install/update_95_xx.php

```diff
@@ -48,6 +48,10 @@ function update95toXX() {
    require __DIR__ . '/update_95_xx/softwares.php';
    include __DIR__ . '/update_95_xx/domains.php';
 
+   // Add unicity key to reservationitem
+   require_once __DIR__ . '/update_95_xx/reservationitem.php';
+   reservationItemAddUnicity($migration);
```

Comment:
I would move this call directly in `/update_95_xx/reservationitem.php`.

Reply:
I've done it this way because I need to pass `$migration` to the function.

Reply:
$migration should be accessible in included file.

By the way, we have to choose which function we use for inclusion :
![image](https://user-images.githubusercontent.com/33253653/96604405-5e1a4700-12f5-11eb-9bb1-9d5218ab4f2f.png)

Reply:
I've switched them to `require` since we don't expect the file to be missing.